src/onegov/api/models.py

```diff
@@ -9,6 +9,7 @@
 from sqlalchemy.orm import relationship
 from uuid import uuid4
 from sqlalchemy import Text
+from typing_extensions import Self
```

Comment:
I would move this import to the `if TYPE_CHECKING:` block. Some of our dependencies will already depend on `typing_extensions`, so it should generally be around at runtime, but we don't directly depend on it ourselves outside of a type checking context, so we should be consistent about that and only use it in type checking context.